services/eventServices.js

```diff
@@ -18,6 +19,12 @@ eventServices.findOneEvent = function(eventId, cb) {
     });
 };
 
+eventServices.findAllEventsInTier = function(givenTier, cb) {
+    Event.find({tier: givenTier}).exec(function(err, event) {
```

Comment:
small detail here, i think we'll want to rename `event` to `events` since we'll be finding multiple events (and will also want to change it when we call the callback function below on line 24)